### DISCUSSION TITLE
Does Seaweed S3 support Lifecycle Expiration ??

### DISCUSSION THREAD

**Discussion by shubham3121** [seaweedfs/seaweedfs#2745] [General]
@chrislusf Can we set auto expiration of incomplete multipart uploads through lifecycle management methods of boto3  in SeaweedFS??

**Comment by guo-sj** [comment]
As far as I known, Seaweedfs haven't supported S3 Lifecycle configuration mechanism yet.

**Reply by shubham3121** [reply]
Thanks, @guo-sj.  Are there any plans to support S3 Lifecycle config mechanism in the current SeaweedFS roadmap ??

**Reply by chrislusf** [reply]
No efficient design yet.

**Comment by atwinand** [comment]
Since the last message is from a year ago, is there any follow up on this? :)

**Comment by TMUniversal** [comment]
I also have a need for this, specifically object expiry, but the other features, such as moving to different availability or another bucket, are of interest to me as well.

**Comment by Trickfilm400** [comment]
based on the documentation, it should be supported?
https://github.com/seaweedfs/seaweedfs/wiki/S3-API-FAQ#setting-ttl

**Reply by chrislusf** [reply]
yes in latest versions. around 4.26~
